examples/cd_chi2ks_adult.ipynb

```diff
@@ -25,7 +25,17 @@
    "cell_type": "code",
```

Comment:
<p>Nitpick: would be good to rerun these sorts of cells without <code>prophet</code>  installed to remove the ugly error in our docs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi-detect/pull/198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>